### PR TITLE
LPS-121642 filter only visible calendars

### PR DESCRIPTION
**Pull request by mrk-vi**
With this changes we avoid throwing errors on calendars fetching when user doesn't have VIEW permission on an invited calendar.

### CODE REVIEW

**Comment by rafaprax** [comment]
ci:test:sf

File: modules/apps/calendar/calendar-service/src/main/java/com/liferay/calendar/service/impl/CalendarBookingServiceImpl.java [hunk 1]
```diff
@@ -338,11 +341,7 @@ public String getCalendarBookingsRSS(
 			calendarBookingLocalService.getChildCalendarBookings(
 				parentCalendarBookingId);
```
Comment (rafaprax) [inline modules/apps/calendar/calendar-service/src/main/java/com/liferay/calendar/service/impl/CalendarBookingServiceImpl.java:343]: Hey @mrk-vi,

Have you tried to use the `filterCalendarBookings(List<CalendarBooking> calendarBookings, String actionId)` method? It seems to have the same intention as your `_filterCalendarBookingsByCalendarVisibility`.

Can you please check this?

Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

File: modules/apps/calendar/calendar-service/src/main/java/com/liferay/calendar/service/impl/CalendarBookingServiceImpl.java (hunk 1 above)
Comment (mrk-vi) [reply modules/apps/calendar/calendar-service/src/main/java/com/liferay/calendar/service/impl/CalendarBookingServiceImpl.java:343]: Hi @rafaprax, 

I've follow your suggestion, but using the method `filterCalendarBookings(List<CalendarBooking> calendarBookings, String actionId)` the result is still not desiderable:

-  an exception is thrown when an user that doesn't have anymore the `VIEW` permission on an invited calendar, try to open his own event.

- also, when an attendee hasn't replied to an invitation yet (status `PENDING`), the `masterEvent` owner can't see the attendee in the list of invitations. So if the organizer update the event, he could accindentally remove the attendee whitout seeing that.

**Comment by rafaprax** [comment]
Just started reviewing :)

**Comment by rafaprax** [comment]
resent here https://github.com/liferay-workflow/liferay-portal/pull/79

Thx for the contribution @mrk-vi

**Pull request closed without merging**